doc/qos_overview.md

```diff
@@ -422,3 +429,176 @@ markings. The same interface has an egress scheduler policy applied to it.
+  # one to group multiple classifiers together and associate them with a
+  # queue.  In concept, the forwarding-group "sends traffic" to a queue.
+  # See the qos data flow diagram to help visualize this.
+  forwarding-groups:
+    - forwarding-group: "input_dest_A"
+      config:
+        name: "input_dest_A"
+        output-queue: dummy_input_queue_A
+    - forwarding-group: "input_dest_B"
+      config:
+        name: "input_dest_B"
+        output-queue: dummy_input_queue_B
+
+  queues:
+    - queue:
+      config:
+        name: "dummy_input_queue_A"
+    - queue:
+      config:
+        name: "dummy_input_queue_B"
+
+  # Here we define 2 scheduler-policies.
+  # The first has a 1Gb policer for traffic on dummy_input_queue_A
+  # The second has a 2Gb policer for traffic on dummy_input_queue_B
+  scheduler-policies:
+    - scheduler-policy:
+      config:
+        name: "limit_1Gb"
+      schedulers:
+        - scheduler:
+          config:
+            sequence: 1
+            type: ONE_RATE_TWO_COLOR
+          inputs:
+            - input: "my input policer 1Gb"
+              config:
+                id: "my input policer 1Gb"
+                input-type: QUEUE
+                queue: dummy_input_queue_A
+          one-rate-two-color:
+            config:
+              cir: 1000000000           # 1Gbit/sec
+              bc: 100000                # 100 kilobytes
+              queuing-behavior: POLICE
+            exceed-action:
+              config:
+                drop: TRUE
+
+    - scheduler-policy:
+      config:
+        name: "limit_2Gb"
+      schedulers:
+        - scheduler:
+          config:
+            sequence: 1
+            type: ONE_RATE_TWO_COLOR
+          inputs:
+            - input: "my input policer 2Gb"
+              config:
+                id: "my input policer 2Gb"
+                input-type: QUEUE
+                queue: dummy_input_queue_B
+          one-rate-two-color:
+            config:
+              cir: 2000000000           # 2Gbit/sec
+              bc: 100000                # 100 kilobytes
+              queuing-behavior: POLICE
+            exceed-action:
+              config:
+                drop: TRUE
+
+  # Here we map interfaces to an input classifer  and input scheduler-policy
+  interfaces:
+    - interface: "PortChannel1.100"
+        config:
+          interface-id: "PortChannel1.100"
+        input:
+          classifers:
+            - classifier:
+              config:
+                name: "dest_A"
+                type: "IPV4"
+            scheduler-policy:
+              state:
+                name: limit_group_A_1Gb
+    - interface: "PortChannel1.200"
+        config:
+          interface-id: "PortChannel1.200"
+        input:
+          classifers:
+            - classifier:
+              config:
+                name: "dest_B"
+                type: "IPV4"
+          scheduler-policy:
+            state:
```

Comment:
Is this a typo?  Should this be a config container?  Ditto below.

Reply:
Yes, typo, fixed these instances